nncf/common/utils/decorators.py

```diff
@@ -0,0 +1,51 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from importlib import import_module
+from typing import Callable, List
+
+from nncf.common.logging import nncf_logger
+
+IMPORTED_DEPENDENCIES = {}
+
+
+def bypass_noreturn_function(dependencies: List[str]) -> Callable:
+    """
+    Decorator to bypass a noreturn function if dependencies are not met.
+
+    :param func: A noreturn function to decorate
```

Comment:
Should probably describe the `dependencies` parameter instead.

Reply:
done